config/jobs/kubernetes/kops/kops-periodics-distros.yaml

```diff
@@ -203,11 +203,11 @@ periodics:
       - --env=KOPS_RUN_TOO_NEW_VERSION=1
       - --extract=release/stable
       - --ginkgo-parallel
-      - --kops-args=--image=ami-02eac2c0129f6376b
+      - --kops-args=
+      - --kops-image=679593333241/CentOS Linux 7 x86_64 HVM EBS ENA 2002_01-b7ee8a69-ee97-4a49-9e68-afaee216db2e-ami-0042af67f8e4dcc20.4
```

Comment:
hm any reason to use the `--kops-image` syntax here as opposed to `--kops-args=--image` everywhere else? its probably worth being consistent one way or another (i dont have a preference either way). and maybe remove the empty `--kops-args` above if we go with `--kops-image`.

Reply:
@justinsb tried using the args for this image but was not very successful. In the end he added a new field for the image. I updated all configs to use `--kops-image` now.